pytest/tests/mocknet/sanity.py

```diff
@@ -0,0 +1,44 @@
+# Connects to the mocknet, issues a tx that sends 100 yoctonear,
+# verifies the balances are updated properly.
+# Deploys a single contracts, ensures it can be invoked.
+
+import sys, time, random, base58
+
+sys.path.append('lib')
+
+from cluster import connect_to_mocknet
+from transaction import sign_payment_tx, sign_function_call_tx, sign_deploy_contract_tx
+from utils import load_binary_file
+
+nodes, accounts = connect_to_mocknet(None)
```

Comment:
A possibly problem here: RpcNode doesn't give you node keys, Did you init keypair else where to sign txn below?

Reply:
Accounts are stored separately from the nodes.
I think generally the idea of coupling nodes and account was weird, I don't want to rewrite the regular infra at this point, but for the new tests I think those should be decoupled.